modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Returns whether or not the string is a valid Hedera account
+ *
+ * @param {string} account - the  account to be validated
+ * @returns {boolean} - the validation result
+ */
+export function isValidAccount(account: string): boolean {
```

Comment:
s/`isValidAccount`/`isValidAddress`/g

Reply:
Great, I've created a new PR for keypair: https://github.com/BitGo/BitGoJS/pull/807
And I am going to be working on these comments there.